go.mod

```diff
@@ -43,6 +43,7 @@ require (
 	go.opentelemetry.io/otel/exporters/trace/jaeger v0.12.0
 	go.opentelemetry.io/otel/sdk v0.12.0
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
+	gotest.tools v2.2.0+incompatible // indirect
```

Comment:
go mod tidy?

Reply:
Yeah, could probably make that apart of CI, along with linting... * shudders *

Reply:
go mod tidy shouldn't be part of ci since it makes permanent modifications to go.mod and go.sum. We should add linting though.

Reply:
I mean adding a step in CI that checks if running `go mod tidy` returns a non-empty diff e.g. what's done in specs-actors here: https://github.com/filecoin-project/specs-actors/blob/master/.circleci/config.yml#L32

Reply:
That would be cool

Reply:
Tracked here: https://github.com/filecoin-project/sentinel-visor/issues/170